app/src/org/commcare/android/resource/installers/XFormAndroidInstaller.java

```diff
@@ -76,8 +77,10 @@ protected int customInstall(Resource r, Reference local, boolean upgrade) throws
 
         FormDef formDef;
         try {
-            formDef = new XFormParser(new InputStreamReader(local.getStream(), "UTF-8")).parse();
-        } catch (XFormParseException xfpe) {
+            XFormParser parser = new XFormParser(new InputStreamReader(local.getStream(), "UTF-8"));

```

Comment:
If possible it might be good to, like, unify in CommCareAndroid "this is where you get an XForm Parser from" so we don't have too keep the registraitons and extensions spread out among multiple places